### PR TITLE
Cdl harvest

### PR DESCRIPTION
**Pull request by mdellabitta**
DO NOT MERGE

Posting this for review from @moltude @markbreedlove.

This seems to work well locally and gets through all of CDL in a reasonable amount of time.

### CODE REVIEW

**Comment by markbreedlove** [comment]
:eyes: ...

File: lib/fetchers/cdl_fetcher.py [hunk 1]
```diff
@@ -1,187 +1,34 @@
...
 
-        if parsed_content is None:
-            error = "Error, there is no content from " \
-                    "URL %s" % url
-        return error, parsed_content
+    def __init__(self, profile, uri_base, config_file):
+        super(CDLFetcher, self).__init__(profile, uri_base, config_file)
+        token = self.config.get("APITokens", "CDL")
+        authorization = \
+            self.http_headers["X-Authentication-Token"].format(token)
```
Comment (markbreedlove) [inline lib/fetchers/cdl_fetcher.py]: Is this supposed to be `profile.get('http_headers')['X-Authentication-Token'].format(token)`?

Comment (mdellabitta) [inline lib/fetchers/cdl_fetcher.py]: I unfortunately was biting off what was there for NYPL. I agree it didn't make much sense, but I think it has something to do with the way http_headers is set by whatever stuff Ingestion1 does to automatically set these properties and that having been written before the idea of auth tokens being passed in from akara.ini started happening. 

You can see what I mean here: https://github.com/dpla/ingestion/blob/develop/lib/fetchers/nypl_fetcher.py#L7-L8

**Comment by moltude** [comment]
I ran this locally and am giving 👍  with a note to add cdl API key to `ingestion-stg` and `ingestion-prod` akara.ini files since those are not managed by ansible

**Pull request merged**